x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -400,9 +400,16 @@ export function registerIndexRoutes({
               ),
             })
           ),
-          model_id: schema.string(),
+          model_id: schema.maybe(schema.string()),
+          pipeline_definition: schema.maybe(
+            // TODO: make this more explicit?
```

Comment:
do we want to track this as an issue or do it now?

Reply:
Good catch! This was slightly lingering (I'd added this when I just had `schema.maybe(schema.any())`), but I did recently find that we had an existing API for providing a pipeline definition, so I copied the schema from [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts#L743-L746), and I feel pretty good about that now. So I've just deleted the TODO.